src/main/commandServer/httpCommandServer.ts

```diff
@@ -62,6 +64,18 @@ export class HttpCommandServer {
   }
 
   async init() {
+    const localHost = '127.0.0.1';
+
+    if (process.platform === 'win32') {
+      this.vtun.addTunnel({
+        name:                  'CLI Server',
+        handshakePort:         17372,
+        vsockHostPort:         17371,
+        peerAddress:           localHost,
+        peerPort:              SERVER_PORT,
```

Comment:
We should have a comment about specifically picking the same port; if we end up choosing a dynamic port on the Windows side we still need a matching number.